container-setup/install/install-rosa.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash -e
+
+if [ "$I_AM_IN_CONTAINER" != "I-am-in-container" ]; then
+  echo "must be run in container";
+  exit 1;
+fi
+
+source /container-setup/install/helpers.sh
```

Comment:
I might be missing something, but is this doing anything?  The only function in there is `remove_coloring()` and I don't see it called/used in here.

Reply:
You're probably right.  ROSA used to be installed via `go get` I believe and that used to dirty up the terminal (because for whatever reason the red color would persist through the rest of the build).  It should be safe to remove this when we move to including only the binaries instead of the entire source repos.